### ISSUE TITLE
Bump the timeout needed to initialise an alba ASD

### ISSUE THREAD

**Issue by jtorreke** [openvstorage/framework-alba-plugin#553]
```
root@srocovsrtr01:~# time mkfs.ext4 -L test /dev/sdb1
mke2fs 1.42.13 (17-May-2015)
Discarding device blocks: done
Creating filesystem with 195352576 4k blocks and 48840704 inodes
*SNIP*
Creating journal (32768 blocks): done
Writing superblocks and filesystem accounting information: done


real    1m11.347s
user    0m0.072s
sys     0m0.132s
```
The current timeout is set to 20 seconds, can we raise this to 300 or so? This is an SSD device.

**Comment by simonOpenV** [comment]
Fixed by #559 
Package information: openvstorage-backend-1.12.0-dev.1518012516.234877a

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/framework-alba-plugin/releases/tag/1.12.0
Packaged in openvstorage-backend-core_1.12.0-1_amd64.deb
